packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -278,6 +278,9 @@ fi
 %endif
 
 %changelog
+* Thu May 20 2021 Feng Xiong <fenxiong@amazon.com> - 1.52.2-1
+- Cache Agent version 1.52.2
+
```

Comment:
Question: should "install script fix: remove dependency on gpg key server." be mentioned in the changelog? Thanks.

Reply:
yeah. added

Reply:
thank you!